### ISSUE TITLE
How to add augmentation to code ?

### ISSUE THREAD

**Issue by minaahmed** [taipingeric/yolo-v4-tf.keras#25]
Hello can you tell where I can add augmentation to the image and labels?
